addons/zeusRecon/functions/fnc_reconStateMachine.sqf

```diff
@@ -0,0 +1,304 @@
+#include "script_component.hpp"
+
+#define RANGE_TO_GROUP 75
+#define TIME_KEEP_SHOWING_AFTER_LOSING_SIGHT 20
+#define ICON_HEIGHT_ABOVE_GROUND 20
+#define RANDOM_GROUP_COLLAPSE_TEST 0.01
+#define REPORT_ICON_MEDICAL "\A3\Ui_f\data\IGUI\RscIngameUI\RscUnitInfo\icon_cross_ca.paa"
+#define REPORT_PRIOITY_MEDICAL 3
+#define REPORT_ICON_RECON "\A3\Ui_F_Curator\Data\Logos\arma3_curator_eye_32_ca.paa"
+#define REPORT_PRIOITY_RECON 1
+
+switch (GVAR(state)) do {
```

Comment:
While an integer switch makes logical sense here, since this is getting called multiple times per frame should this be in the if () exitWith {}; form? It's supposed to be minorly more performant